modules/page/html/render/private.js

```diff
@@ -19,7 +19,10 @@ exports.create = function (api) {
     const i18n = api.intl.sync.i18n
     var id = api.keys.sync.id()
     var compose = api.message.html.compose({
-      meta: {type: 'post'},
+      meta: {
+        type: 'post',
+        location: `${path}/${id}`
```

Comment:
@KGibb8 

Hmm, probably the `location` shouldn't be inside of meta. Should probably just be in the root options passed to `compose`. Otherwise it will be included in the final published message.

This is getting real close!

Reply:
Whoops! Ofc! :man_facepalming: